dotcom-rendering/src/web/components/SlotBodyEnd.importable.tsx

```diff
@@ -59,12 +59,18 @@ const buildBrazeEpicConfig = (
 	brazeMessages: BrazeMessagesInterface,
 	countryCode: string,
 	idApiUrl: string,
+	contentType: string,
```

Comment:
I can see that it is already in use in this component but it's worth keeping in mind that the preferred method for branching logic like this is to use `format`, properties like `contentType` are still in use and do still work but this could change in the future.